notebooks/official/model_evaluation/automl_tabular_classification_model_evaluation.ipynb

```diff
@@ -8,7 +8,7 @@
       },
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #25.        from google_cloud_pipeline_components.experimental.evaluation import (
</pre><p>Can you change imports to this?</p><p>from google_cloud_pipeline_components.v1.model_evaluation import ModelEvaluationClassificationOp</p><p>from google_cloud_pipeline_components.preview.evaluation import ModelEvaluationFeatureAttributionOp</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/2038/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='64'/>

Reply:
done